java/kafka/common/src/main/java/TracingSystem.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+public enum TracingSystem {
```

Comment:
This seems like a huge overkill to have a module just for a single enum. I would probably integrate it into the modules directly. (But maybe it should have a further discussion since my guess is that this is how it should be done :-o).

Are there at least any plans to add more stuff to it in the next PRs?